docs/spec.md

```diff
@@ -4724,8 +4724,8 @@ undefined.
       `scatter_dims_to_operand_dims`[i] $\lt$ rank(`inputs`[0]).
 * (C14) $0 \le$ `index_vector_dim` $\le$ rank(`scatter_indices`).
 * (C15) `update_computation` has type `(tensor<E0>, ..., tensor<EN-1>, tensor<E0>, ..., tensor<EN-1>) -> (tensor<E0>, ..., tensor<EN-1>)`
-        where `Ek = element_type(inputs[k])` for any k $\in$ [0, N).
-* (C16) `inputs[k]` and `result[k]` have the same type for any k $\in$ [0, N).
+        where `Ek = element_type(inputs[k])` for all k $\in$ [0, N).
```

Comment:
In C13 there seems to have a typo
`i $\in$ [0, size(`scatter_dims_to_operand_dims`))` should be
`i $\in$ [0, size(`scatter_dims_to_operand_dims`)]`

Reply:
```
(C13) For all i $\in$ [0, size(`scatter_dims_to_operand_dims`)), $0 \le$
      `scatter_dims_to_operand_dims`[i] $\lt$ rank(`inputs`[0]).
```
Correct me if I'm wrong, but since the `i` indexes into `scatter_dims_to_operand_dims`, I think the current formulation is correct.

Reply:
That is right. I misinterpreted. never mind.